en/docs/using-wso2-identity-server/authentication-session-persistence.md

```diff
@@ -132,39 +131,21 @@ before the other node accesses the same data from the database.
     not triggered when adding new data.
     
 
-The following configuration found in the
-`         <IS_HOME>/repository/conf/identity/identity.xml        ` file,
-under the the `         Server        ` and
-`         JDBCPersistenceManager        ` elements is used to enable
-session persistence.
+Add the following configuration the
```

Comment:
this should be "Add the following configuration **to** the....."

Reply:
Done.